src/chains/ethereum/ethereum/src/api.ts

```diff
@@ -2931,11 +2931,12 @@ export default class EthereumApi implements Api {
    * console.log(transactionTrace);
    * ```
    */
+  @assertArgLength(1, 2)
```

Comment:
I noticed this was missing from both `debug_` methods so I added it. I can move it to another PR if anyone objects to it being in here.